packages/tables/resources/views/index.blade.php

```diff
@@ -400,15 +412,35 @@ class="shrink-0"
                             >
                                 @if ($isReordering)
                                     <x-tables::reorder.cell />
-                                @elseif ($isSelectionEnabled)
-                                    <x-tables::checkbox-cell>
-                                        <x-slot
-                                            name="checkbox"
-                                            x-model="selectedRecords"
-                                            :value="$recordKey"
-                                            class="table-row-checkbox"
-                                        ></x-slot>
-                                    </x-tables::checkbox-cell>
+                                @else
+                                    @if (count($actions) && $actionsPosition === Position::BeforeCells)
+                                        <x-tables::actions-cell
```

Comment:
I'm wondering if we could abstract this, since we repeat it 3 times here now.

Reply:
I was wondering that too

Reply:
Maybe move the `wire:loading` and `wire:target` into the actions cell component? 🤔

Reply:
Maybe move the `wire:loading` and `wire:target` into the actions cell component? 🤔

Reply:
Done